src/watch.py

```diff
@@ -0,0 +1,37 @@
+import time
+from pathlib import Path
+
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+
+class Handler(FileSystemEventHandler):
```

Comment:
I think a better API here would be to provide a callback to the handler, called when something happens on the filesystem. It would be easier to test too.
I'll do it when I plug this part with the exercise execution logic.

Reply:
TODO